testlib/src/test/java/com/diffplug/spotless/go/GofmtFormatStepTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2024 DiffPlug
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.diffplug.spotless.go;
+
+import org.junit.jupiter.api.Test;
+
+import com.diffplug.spotless.ResourceHarness;
+import com.diffplug.spotless.StepHarness;
+import com.diffplug.spotless.tag.GofmtTest;
+
+@GofmtTest
+public class GofmtFormatStepTest extends ResourceHarness {
+	@Test
+	void test() {
+		try (StepHarness harness = StepHarness.forStep(GofmtFormatStep.withVersion("go1.21.5").create())) {
```

Comment:
It should be only `1.21.5`?

Reply:
This is the version I have installed on my machine. Since special tests are not executed on CI, I thought that any available version would be sufficient here.

Reply:
Got it.